src/main/java/ua/danit/final_project/dto/LocationDto.java

```diff
@@ -0,0 +1,15 @@
+package ua.danit.final_project.dto;
+
+import lombok.Data;
+
+import java.io.Serializable;
+import java.util.List;
+
+@Data
+public class LocationDto implements Serializable {
+
+  private Long id;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "id" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=131&issues=AWY0HPGCM63is4RD-5TZ&open=AWY0HPGCM63is4RD-5TZ)

[ISSUE_KEY]: AWY0HPGCM63is4RD-5TZ